sky/adaptors/fluidstack.py

```diff
@@ -0,0 +1,29 @@
+"""fluidstack cloud adaptor."""
+
+import functools
+
+_fluidstack_sdk = None
+
+
+def import_package(func):
+
+    @functools.wraps(func)
+    def wrapper(*args, **kwargs):
+        global _fluidstack_sdk
+        if _fluidstack_sdk is None:
+            try:
+                import fluidstack as _fluidstack  # pylint: disable=import-outside-toplevel
```

Comment:
We should update the `setup.py` to have an extra for `fluidstack` that installs the dependency.

Reply:
Hmm, actually, it seems we are not using `fluidstack` package anywhere. Should we leave this file out?